include/lib/stdlib/sys/endian.h

```diff
@@ -25,6 +25,11 @@
  *
  * $FreeBSD$
  */
+/*
+ * Copyright (c) 2017, ARM Limited and Contributors. All rights reserved.
```

Comment:
Actually, it doesn't look like you've changed this file at all so no need to add the "ARM Limited and Contributors" copyright.